src/lib/functions/registry.js

```diff
@@ -143,7 +144,10 @@ class FunctionsRegistry {
     log(`${NETLIFYDEVLOG} ${chalk.green('Loaded')} function ${chalk.yellow(name)}.`)
   }
 
-  async scan(directories) {
+  async scan(relativeDirs) {
+    const directories = relativeDirs.filter(Boolean).map((dir) => (isAbsolute(dir) ? dir : join(this.projectRoot, dir)))
```

Comment:
@erezrokah @ehmicky I think this is the fastet way to fix it -> Wondering if we can fix it on the long term inside the netlify config.

Reply:
Yes, this should be address by https://github.com/netlify/build/issues/3620.